packages/block-editor/src/components/block-preview/index.js

```diff
@@ -20,11 +21,41 @@ import { BlockListItems } from '../block-list';
 
 export function BlockPreview( {
 	blocks,
-	__experimentalPadding = 0,
 	viewportWidth = 1200,
+	minHeight,
+	additionalStyles = [],
+	// Deprecated props:
 	__experimentalMinHeight,
-	__experimentalStyles = [],
+	__experimentalStyles,
+	__experimentalPadding,
 } ) {
+	if ( __experimentalMinHeight ) {
+		minHeight = __experimentalMinHeight;
+		deprecated( 'The __experimentalMinHeight prop', {
+			since: '6.2',
+			alternative: 'minHeight',
+		} );
+	}
+	if ( __experimentalStyles ) {
+		additionalStyles = __experimentalStyles;
+		deprecated( 'The __experimentalStyles prop of BlockPreview', {
+			plugin: 'Gutenberg',
+			since: '15.0',
+			version: '15.2',
+			alternative: 'additionalStyles',
+		} );
+	}
+	if ( __experimentalPadding ) {
```

Comment:
I think we should leave the default value of `0` as before, no? Maybe add it as default value inside `additionalStyles`? 🤔

Reply:
It's not the same because before with `0` as a default, we always added it: `bodyElement.style.padding = __experimentalPadding + 'px';`. That means that if the theme have `padding` for the body in theme.json it will be used in the previews now.

Reply:
Aha! Thanks for patiently explaining. I'm with you now. In 8a3838a4e1 I added `padding: 0` to the default styles and changed the order of `styles` such that `additionalStyles` can override the default styling. That make sense?

Reply:
Thanks Robert!